irohad/torii/processor/impl/stub_query_processor.cpp

```diff
@@ -25,7 +25,9 @@ namespace iroha {
     using dao::QueryResponse;
     using dao::Client;
 
-    QueryProcessorStub::QueryProcessorStub() {
+    QueryProcessorStub::QueryProcessorStub(ametsuchi::WsvQuery &wsv,
+                                           ametsuchi::BlockQuery &block) :
```

Comment:
Yes. Of course, AmetsuchiStub methods don't need.
I agree that QueryProcessorStub has `ametsuchi::WsvQuery wsv_` and `ametsuchi::BlockQuery block_`.

I mean:
`iroha::torii::QueryProcessorStub qp(ametsuchi, ametsuchi);`
can be changed to:
`iroha::torii::QueryProcessorStub qp(ametsuchi);`

but it's minor things, it can be ignored.